typescript/agentkit/CHANGELOG.md

```diff
@@ -54,6 +54,7 @@
 
 ### Added
 
+- Added `openseaActionProvider` to list NFTs on OpenSea.
```

Comment:
Please remove the manual changelog entry. We now use changesets to generate changelogs.

Follow [this README](https://github.com/coinbase/agentkit/blob/main/CONTRIBUTING-TYPESCRIPT.md#changelog) and generate the changeset via `npx @changesets/cli`